proto/storage/alert.proto

```diff
@@ -126,7 +126,7 @@ message Alert {
     repeated Violation        violations        = 5  [(gogoproto.moretags) = 'sensorhash:"ignore" search:"-"'];
     ProcessViolation          process_violation = 13 [(gogoproto.moretags) = 'search:"-"'];
 
-    Enforcement               enforcement       = 6 [(gogoproto.moretags) = 'search:"Enforcement"'];
+    Enforcement               enforcement       = 6;
```

Comment:
Enforcement is a message so this is actually unsupported and caused the search walker to crash